var/spack/repos/builtin/packages/py-ucsf-pyem/package.py

```diff
@@ -33,8 +36,35 @@ class PyUcsfPyem(PythonPackage):
     depends_on("py-healpy@1.11:", type=("build", "run"))
     depends_on("py-natsort@6.0:", type=("build", "run"))
 
-    # Extracts files into bin for proper useage
+    script_list = [
```

Comment:
This list may change from one version to the next. May be easier to use a glob and then an if-statement to skip setup.py.

Reply:
That would indeed be more sensible!